pkg/server/endpoints/bundle/internal/acmetest/ca.go

```diff
@@ -33,7 +33,7 @@
 //   appropriately by the SPIRE KeyManager signers.
 // - Fails new-reg requests if the terms-of-service has not been accepted
 
-//nolint // forked code
+// nolint // forked code
```

Comment:
This change needs to be reverted. `//nolint` (without the space) is the required syntax for the linter to recognize the directive.

Reply:
the editor setting was messing this up both in vi and vscode with go fmt